README.md

```diff
@@ -1 +1,238 @@
 # Sync and Async
+
+## Overview
+
+We've spent a lot of time in the beginning of Unit 1 talking about non-blocking input/output. While this pattern is great for performance, sometimes you will need to write synchronous code in Node.
+
+Yes, synchronous code might be useful. Imagine a situation where you read configurations from a file to boot up your system. Without the configurations, the system won't work. In this case, you don't want asynchronous code. The synchronous pattern would be a better idea, because the system won't start without the configurations.
+
+On the other hand, one of the major challenges in learning Node.js is its asynchronous code patterns. Human brain just didn't evolve to think in parallel processes. This hurdle is exacerbated when people learning Node are experienced developers in other more traditional and synchronous languages (PHP, Java, Ruby, Python, C).
+
+Understanding asynchronous code and its patterns is the key to mastering Node.js. Besides the async code, and a few differences like module system, globals and data types, Node is almost the same as browser JavaScript. For this reason, let's dig deeper into the async code and how we can implement it.
+
+This lesson will cover the sync and async code by using `fs` as an example.
+## Objectives
+
+1. Illustrate sync code with `fs` file read method
```

Comment:
"Illustrate" is a really broad word. Can we find a better way to clue students in on what they'll be able to accomplish? We're trying to give them tools, I think, to help them evaluate when to use synchronous or asynchronous code in Node.js.